lib/screen/clubs/clubs_screen.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_template/styles/theme_dimens.dart';
+import 'package:flutter_template/viewmodel/clubs/clubs_viewmodel.dart';
+import 'package:flutter_template/widget/clubs/club_card.dart';
+import 'package:flutter_template/navigator/mixin/back_navigator.dart';
+import 'package:flutter_template/navigator/mixin/error_navigator.dart';
+import 'package:flutter_template/widget/provider/provider_widget.dart';
+import 'package:get_it/get_it.dart';
+
+class ClubsScreen extends StatefulWidget {
+  static const routeName = 'club_selection';
+  const ClubsScreen({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  ClubsScreenState createState() => ClubsScreenState();
+}
+
+@visibleForTesting
+class ClubsScreenState extends State<ClubsScreen>
+    with BackNavigatorMixin, ErrorNavigatorMixin
+    implements ClubsViewNavigator {
+  @override
+  Widget build(BuildContext context) {
+    return ProviderWidget<ClubsViewModel>(
+      create: () => GetIt.I()..init(this),
+      consumerWithThemeAndLocalization:
+          (context, viewModel, child, theme, localization) {
+        return Scaffold(
+          backgroundColor: theme.colorsTheme.backgroundDark,
+          body: Center(
+            child: LayoutBuilder(builder: (context, constraints) {
+              if (constraints.maxWidth > 1100) {
```

Comment:
create a custom widget that handles your breakpoints. (ResponseWidget could be used maybe)